Changelog.md

```diff
@@ -22,6 +22,7 @@ Compiler Features:
  * Yul Optimizer: Try to simplify function names.
  * Yul IR Generator: Report source locations related to unimplemented features.
  * Optimizer: Optimize ``exp`` when base is 0, 1 or 2.
+ * Optimizer: Optimize ``exp`` when base is -1.
```

Comment:
Maybe merge with the entry above, if still same release?

Reply:
needs to move to the new version.